uniflow/flow/transform/model.py

```diff
@@ -197,7 +105,7 @@ def _deserialize(self, data: List[str]) -> List[Dict[str, Any]]:
         }
 
 
-class PreprocessModel(Model):
+class LLMDataPreprocessor(LLMDataProcessor):
```

Comment:
same as above, this name is too long.

Reply:
For scientists, it's not too long but clear enough.